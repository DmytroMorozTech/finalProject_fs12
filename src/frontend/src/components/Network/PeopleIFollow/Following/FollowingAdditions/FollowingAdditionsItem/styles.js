import { makeStyles } from '@material-ui/core/styles'

export default makeStyles((theme) => ({

  item: {
    display: 'flex',
    flexDirection: 'column',
    cursor: 'pointer'
  },

  title: {
    fontSize: theme.typography.h5.fontSize,
    fontWeight: theme.typography.fontWeightBold
  },

  subtitle: {
    color: theme.palette.grey[600],
    fontSize: theme.typography.h6.fontSize
  }
}))