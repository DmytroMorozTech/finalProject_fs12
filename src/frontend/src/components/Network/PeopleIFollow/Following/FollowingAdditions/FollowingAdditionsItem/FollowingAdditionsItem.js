import styles from './styles'
import React from 'react'

function FollowingAdditionsItem ({ title, subtitle, onClick }) {
  const classes = styles()

  return (
    <div className={classes.item} onClick={onClick}>
      <div className={classes.title}>{title}</div>
      <div className={classes.subtitle}>{subtitle}</div>
    </div>
  )
}

export default FollowingAdditionsItem
