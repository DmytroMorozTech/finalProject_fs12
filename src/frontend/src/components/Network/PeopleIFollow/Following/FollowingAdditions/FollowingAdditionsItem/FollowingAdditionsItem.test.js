import { shallow } from 'enzyme'
import React from 'react'
import FollowingAdditionsItem from './FollowingAdditionsItem'

jest.mock('./styles', () => () => ({}))

const setUp = (props) => shallow(<FollowingAdditionsItem {...props}/>)

describe('should render FollowingAdditionsItem component', () => {
  let component
  let mockFunc = jest.fn()
  beforeEach(() => {
    component = setUp({
      title: 'All',
      subtitle: '',
      onClick: mockFunc
    })
  })
  it('to match snapshot', () => {
    expect(component).toMatchSnapshot()
  })
})
