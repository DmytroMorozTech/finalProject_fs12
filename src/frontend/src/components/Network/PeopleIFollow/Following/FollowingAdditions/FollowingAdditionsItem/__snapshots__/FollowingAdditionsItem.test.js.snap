// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render FollowingAdditionsItem component to match snapshot 1`] = `
<WithStyles(ForwardRef(MenuItem))
  onClick={[MockFunction]}
>
  <div>
    All
  </div>
  <div />
</WithStyles(ForwardRef(MenuItem))>
`;
